contracts/data-storage/andromeda-primitive/src/testing/tests.rs

```diff
@@ -81,6 +81,47 @@ fn test_set_and_update_value_without_key() {
     );
 }
 
+#[test]
+fn test_set_value_invalid() {
+    let (mut deps, info) = proper_initialization(PrimitiveRestriction::Private);
```

Comment:
Mind writing this in the `TestCase` structure so we can add future cases quickly?